gomint-server/src/main/java/io/gomint/server/command/CommandManager.java

```diff
@@ -239,7 +239,7 @@ private CommandOutput tryCommandDispatch( CommandSender sender, CommandHolder co
             return command.getExecutor().execute( sender, command.getName(), arguments );
         } catch ( Exception e ) {
             LOGGER.warn( "Command '{}' failed", command.getName(), e );
-            return new CommandOutput().fail( "Command has thrown an error. Please check the logs" );
+            return CommandOutput.failure( e );
```

Comment:
This should never happen. Errors are not meant for the client. This can contain sensible data like database queries, passwords, private keys or IPs etc.